src/CoreData/Enums.cs

```diff
@@ -58,7 +58,9 @@ public enum NSAttributeType : ulong {
 		[MacCatalyst (13, 1)]
 		Uri = 1200,
 		Transformable = 1800,
-		ObjectID = 2000
+		ObjectID = 2000,
+		[Watch (7, 0), TV (14, 0), Mac (11, 0), iOS (14, 0), MacCatalyst (17, 0)]
```

Comment:
Why old versions here?

https://developer.apple.com/documentation/coredata/nsattributetype/compositeattributetype?changes=__6

Reply:
due to a yy.